TMBuilder/src/main/java/com/ebay/tool/thinmodelgen/gui/menu/TMBuilderMenu.java

```diff
@@ -366,7 +373,12 @@ private void doReviewValidationSet(String validationSetName) {
         }
       }
 
-      String message = new ThinModelExport().getValidationStatementsForValidationSet(setModel);
+      String message = "";
+      if(RuntimeConfigManager.getInstance().getPlatform().equals(Platform.ANDROID))
```

Comment:
RuntimeConfigManager only applies to NST and won't apply to the TMBuilder. This is going to have to be changed to not depend on RuntimeConfigManager.

Reply:
Ah I see, how do we get to choose which one to output for Kotlin or Java ?

Reply:
I was able to achieve it by using selected file format,